dpgen/auto_test/Gamma.py

```
@@ -472,12 +472,15 @@
                 )
 
                 miller_index = loadfn(os.path.join(ii, "miller.json"))
-                ptr_data += "%-25s     %7.2f   %7.3f    %8.3f %8.3f\n" % (
-                    str(miller_index) + "-" + structure_dir + ":",
-                    int(ii[-4:]) / self.n_steps,
-                    sfe,
-                    epa,
-                    equi_epa_slab,
+                ptr_data += (
+                    "%-25s     %7.2f   %7.3f    %8.3f %8.3f\n"
```

Ruff (0.8.0)

Use format specifiers instead of percent format

(UP031)